src/api/sendGrid.ts

```diff
@@ -0,0 +1,23 @@
+const sgMail = require('@sendgrid/mail');
+const template_id = 'd-2e3aabdb9a3c45a393877596534c57c2'; // temporary holder, change to required template id
+
+export interface Personalizations {
+    to: string|string[];
+    dynamic_template_data?: Record<string, string>;
+}
+
+export const sendEmail = async (personalizations: Personalizations[]) => {
+
+    sgMail.setApiKey(process.env.SENDGRID_API_KEY);
```

Comment:
1. move it to outside of this function
2. don't call process.env directly. use apiEnv instead. refer to other callers of apiEnv.